src/main/resources/static/css/style.css

```diff
@@ -2944,32 +2944,23 @@ input:checked+.form-slider:before {
 	display: none;
 }
 
-.animate
-{
+.animate {
 	transition: all 0.1s;
-	-webkit-transition: all 0.1s;
 }
 
-.action-button
-{
-	position: relative;
+.action-button {
 	padding: 5px 10px;
 	margin: 0px 5px 5px 0px;
-	float: right;
 	border-radius: 5px;
-	color: #FFF;
-	text-decoration: none;
+	color: white;
 }
 
-.action-button:active
-{
+.action-button:active {
 	transform: translate(0px,5px);
-	-webkit-transform: translate(0px,5px);
 	border-bottom: 1px solid;
 }
```

Comment:
The bouncing of the document persists. Right now you do this, upon a click:
translate on x down by 5px, and decrease `border-bottom` to 1px (`border-bottom` originally in grey-button is 5px).
No problem with that. But you decrease the height of the element by 4px. So increase the `margin-bottom` in this class with 4px.
Somewhat like this:
```css
.action-button:active {
	transform: translate(0px,5px);
	border-bottom: 1px solid;
	margin: 5px 0px 9px 0px; /* see, I've increased 5 to 9 */
}
```